### PR TITLE
Applied PHP CS Fixer

### PR DESCRIPTION
**Pull request by BelleNottelling**
Non functional change, really just gets the code styling to be consistent

### CODE REVIEW

File: src/bb-modules/Servicecentovacast/ccapiclient.php [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
  *
- * Note that all of the methods defined in the classes below should be
- * considered private; method overloading is used to dynamically handle
- * calls to what would be the public methods of each class.
+ * Copyright FOSSBilling 2022
+ * This software may contain code previously used in the BoxBilling project.
+ * Copyright BoxBilling, Inc 2011-2021
  *
+ * This source file is subject to the Apache-2.0 License that is bundled
+ * with this source code in the file LICENSE
```
Comment (evrifaessa) [inline src/bb-modules/Servicecentovacast/ccapiclient.php:14]: Seems like a problem to me.

File: src/bb-modules/Servicesolusvm/SolusVM.php [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+ * @license   Apache-2.0
+ *
+ * Copyright FOSSBilling 2022
+ * This software may contain code previously used in the BoxBilling project.
+ * Copyright BoxBilling, Inc 2011-2021
+ *
+ * This source file is subject to the Apache-2.0 License that is bundled
+ * with this source code in the file LICENSE
+ */
+
```
Comment (evrifaessa) [inline src/bb-modules/Servicesolusvm/SolusVM.php:16]: This one too. Maybe we shouldn't automatically apply headers?

File: src/bb-modules/Spamchecker/akismet.curl.class.php [hunk 3]
```diff
@@ -1,54 +1,17 @@
...
- * @author     David Pennington <@codexplorer.com>
- * @copyright  Copyright (c) 2008 CodeXplorer <http://www.codexplorer.com>
- * @license    http://www.gnu.org/licenses/gpl-3.0.html (GPL v3)
+ * Copyright FOSSBilling 2022
+ * This software may contain code previously used in the BoxBilling project.
+ * Copyright BoxBilling, Inc 2011-2021
  *
- ********************************** 80 Columns *********************************
+ * This source file is subject to the Apache-2.0 License that is bundled
+ * with this source code in the file LICENSE
```
Comment (evrifaessa) [inline src/bb-modules/Spamchecker/akismet.curl.class.php:14]: This too.

**Comment by BelleNottelling** [comment]
Fixed those headers, good catch

File: src/bb-modules/Servicesolusvm/SolusVM.php (hunk 2 above)
Comment (BelleNottelling) [reply src/bb-modules/Servicesolusvm/SolusVM.php]: > Maybe we shouldn't automatically apply headers?

I think it's fine, I just added exclusions for those folders & most things we add that isn't our code are through composer these days so this won't even effect that

**Review by evrifaessa — APPROVED** [review]

**Review by John-S4 — APPROVED** [review]
Anything that says 174 files changed is a bit terrifying, but all seems good to me.

**Comment by BelleNottelling** [comment]
> Anything that says 174 files changed is a bit terrifying, but all seems good to me. 

Thankfully they are all stylistic changes

**Pull request merged**